src/main/java/org/mastodon/mamut/launcher/MastodonLauncher.java

```diff
@@ -654,4 +634,49 @@ private WindowManager createWindowManager()
 	{
 		return new WindowManager( context );
 	}
+
+	private void openMastodonMainWindow( @Nonnull File file )
```

Comment:
This methods name is misleading, it does more than opening the Mastodon main window.
It also closes the launcher, and it adds the project to the list of recent projects.

Reply:
I agree.
I kept the original `loadMastodonProject`-method, since in another test, it worked also with this method.